src/js/otus/hooks.ts

```diff
@@ -0,0 +1,38 @@
+import { useFetchOTU } from "@otus/queries";
+import { OTUSequence } from "@otus/types";
+import { indexOf, map, sortBy } from "lodash-es/lodash";
+
+/**
+ * A hook for sorting the sequences for the active isolate
+ *
+ * @param otuId - The otu which the isolate belongs to
+ * @param sequences - The sequences in the active isolate
+ */
+export default function useGetSequences(otuId: string, sequences: OTUSequence) {
+    const { data, isLoading } = useFetchOTU(otuId);
+
+    if (isLoading) {
+        return { sequences: [], isLoading: true };
+    }
```

Comment:
Couple generic comments that I think could help clean this up a lot: 

1. There isn't really a need to manually define the value for isLoading since what the query returns is always correct
2. I would treat the API of this hook as if it was a more general query. ie before data is loaded, data can be undefined a per a normal query

Combing these two I am guessing you should be able to get it down to one return, assuming that post query success sequences is at least an empty array